hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/README.md

```diff
@@ -21,7 +21,12 @@ The YARN UI is an Ember based web-app that provides visualization of the applica
 
 ## Configurations
 
-* You can point the UI to custom locations by setting the environment variables in `src/main/webapp/config/configs.env`
+You can point the UI to custom locations by setting the environment variables in `src/main/webapp/config/configs.env`.
+
+In order to access RM from UI started by `yarn start`,
```

Comment:
"from the UI"? I'm not sure what we are trying to say here.
I would recommend rephrasing this.

Reply:
addressed by adding the development context.